### PR TITLE
fix: Attach comma token to MATCH_ARM instead of MATCH_ARM_LIST

### PR DESCRIPTION
**Pull request by Veykril**
bors r+

### CODE REVIEW

**Comment by Veykril** [comment]
bors retry

File: crates/parser/src/grammar/expressions/atom.rs [hunk 1]
```diff
@@ -452,8 +439,21 @@ fn match_arm(p: &mut Parser) -> BlockLike {
...
+    //         _ => (),
+    //         _ => {}
+    //         _ => ()
+    //     }
+    // }
+    if blocklike.is_block() {
+        p.eat(T![,]);
+    } else if !p.at(T!['}']) {
+        p.expect(T![,]);
+    }
```
Comment (matklad) [inline crates/parser/src/grammar/expressions/atom.rs:455]: Can be simplified to eating , unconditionally, and then failing if not eof or blocklike

**Comment by matklad** [comment]
👍 

I think we should apply this treatment to all nodes. That is, attach commas to fields, parameters, arguments, etc. that’s going to be annoying and invasive change, but I think one we need to do (and one that’s been burning at the back of my mind forever).

we should do it one element at the time, and this also needs ungrammar updates.

**Comment by Veykril** [comment]
This one here doesn't need an ungrammar change fortunately

bors r+

**Comment by Veykril** [comment]
bors retry

**Pull request merged**